core/client.go

```diff
@@ -302,73 +292,13 @@ func (c *Client) reconnect(ctx context.Context, addr string) {
 	}
 }
 
-func (c *Client) init() {
-	// // tracing
-	// _, _, err := tracing.NewTracerProvider(c.name)
-	// if err != nil {
-	// 	logger.Errorf("tracing: %v", err)
-	// }
-}
-
-// ServerAddr returns the address of the server.
-func (c *Client) ServerAddr() string {
-	return c.addr
-}
-
-// initOptions init options defaults
-func (c *Client) initOptions() error {
-	// logger
-	if c.logger == nil {
-		if c.opts.Logger != nil {
-			c.logger = c.opts.Logger
-		} else {
-			c.logger = logger.Default()
-		}
-	}
-	// observe tag list
-	if c.opts.ObserveDataTags == nil {
-		c.opts.ObserveDataTags = make([]frame.Tag, 0)
-	}
-	// credential
-	if c.opts.Credential == nil {
-		c.opts.Credential = auth.NewCredential("")
-	}
-	// tls config
-	if c.opts.TLSConfig == nil {
-		tc, err := pkgtls.CreateClientTLSConfig()
-		if err != nil {
-			c.logger.Errorf("%sCreateClientTLSConfig: %v", ClientLogPrefix, err)
-			return err
-		}
-		c.opts.TLSConfig = tc
-	}
-	// quic config
-	if c.opts.QuicConfig == nil {
-		c.opts.QuicConfig = &quic.Config{
-			Versions:                       []quic.VersionNumber{quic.Version2},
-			MaxIdleTimeout:                 time.Second * 40,
-			KeepAlivePeriod:                time.Second * 20,
-			MaxIncomingStreams:             1000,
-			MaxIncomingUniStreams:          1000,
-			HandshakeIdleTimeout:           time.Second * 3,
-			InitialStreamReceiveWindow:     1024 * 1024 * 2,
-			InitialConnectionReceiveWindow: 1024 * 1024 * 2,
-			TokenStore:                     quic.NewLRUTokenStore(10, 5),
-			// DisablePathMTUDiscovery:        true,
-		}
-	}
-	// credential
-	if c.opts.Credential != nil {
-		c.logger.Printf("%suse credential: [%s]", ClientLogPrefix, c.opts.Credential.Name())
-	}
-
-	return nil
-}
+// Addr returns the address of the client.
+func (c *Client) Addr() string { return c.addr }
```

Comment:
Why change the `ServerAddr()` to `Addr()`?

Reply:
This function will be called in the form of `client.ServerAddr()`, that make my confusion.